src/index.html

```diff
@@ -545,8 +545,17 @@
     <div class="subheader">Tooltips</div>
     <div class="tooltip-container">
       <div class="tooltip tooltip--active">
-        <div class="tooltip__icon">i</div>
-        <div class="tooltip__content">
+        <div class="tooltip__icon">
+           <svg width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26.42 26.42">
+                <defs><style>.cls-1{fill:#667485;}</style></defs>
```

Comment:
please remove style block and set the fill inline so we can remove all class selectors.